qcodes/dataset/measurements.py

```diff
@@ -657,7 +661,8 @@ def __exit__(self, exception_type, exception_value, traceback) -> None:
             # and finally mark the dataset as closed, thus
             # finishing the measurement
             self.ds.mark_completed()
-            log.info(f'Finished measurement with guid: {self.ds.guid}')
+            log.info(f'Finished measurement with guid: {self.ds.guid}. '
```

Comment:
Do we need to repeat the log info here? It already has the same info on the `starting measurement` part with the matching guid.

Reply:
For the telemerty plot shown below, we need both the start and the end time of the measurement. That is why we have the repitition. 

![image](https://user-images.githubusercontent.com/18750964/68387789-6ff3f000-015f-11ea-8f98-bd16ba48eeef.png)

Reply:
Sorry for the misunderstanding. Reading my comment again, shows myself that I really have not done a good job in communicating ;-). I was only referring to the extra log info, that you implement here.

Reply:
Dominik, that is correct. Once I have the GUID from start measurement this should not be needed.

Reply:
The offline discussion happened between Dominik, Mikhail and me, and we agreed that It is ok to keep the extra_log_info in the finished measurement for the added convenience in handling data in appinsights and powerBI